addons/advanced_ballistics/functions/fnc_handleFired.sqf

```diff
@@ -88,7 +88,7 @@ if (GVAR(ammoTemperatureEnabled) || GVAR(barrelLengthInfluenceEnabled)) then {
 
 _bulletTraceVisible = false;
 if (GVAR(bulletTraceEnabled) && cameraView == "GUNNER") then {
-    if (currentWeapon ACE_player in ["ACE_Vector", "Binocular", "Rangefinder", "Laserdesignator"]) then {
+    if ((currentWeapon ACE_player == binocular ACE_player) && {binocular ACE_player != ""}) then {
```

Comment:
I was thinking of a civilian with no weapon: `"" == ""`, but I guess they wouldn't be able to be in "GUNNER" view